Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -418,29 +422,41 @@ function Get-MachineNameFromId
           [System.Collections.Hashtable]$map,
           [string]$mapParameter,
           [Object]$azureRMVMResources,
-          [boolean]$throwOnTotalUnavaialbility)
+          [boolean]$throwOnTotalUnavailability,
+          [string]$debugLogsFlag)
 
     if($map)
     {
+        if($debugLogsFlag -eq "true")
+        {
+            Write-Verbose "Map for $mapParameter : " -Verbose
```

Comment:
Synced up with pavan before making porting, whether to retain verbose flag or not. And according to him we should retain as changes for not using verbose might not be in m95 branch.